### DISCUSSION TITLE
enroll 2nd car

### DISCUSSION THREAD

**Discussion by dzarth** [llamafilm/tesla-http-proxy-addon#26] [General]
Thanks a lot for this addon! I managed to get it working for my car, but I'm struggling how to enroll a public key in my wife's vehicle as the addon Web UI is disabled now.

Many thanks in advance,
Dirk

**Comment by llamafilm** [comment]
Open this link on your iPhone: https://www.tesla.com/_ak/tesla.example.com (replacing the domain)

**Reply by dzarth** [reply]
I found that URL in the addons index.html and tried it first. It would not let me pass the "finish setup" dialog. Now where you mentioned it, I tried again, rebooted my pixel, tried an other phone with the tesla app installed and logged in but still no luck.

**Reply by dzarth** [reply]
ok, I managed it. For Android at least it must be a clickable link on a webpage. Entering the URL manually in a browser wouldn't work. This way it functioned flawlessly with the Web UI.
Again, thanks a lot for your work. I'm looking forward to automate charging on solar now :)

**Reply by llamafilm** [reply]
Thanks for the feedback.  I'll add a note about this in the docs.  Do you know how it works if you have 2 vehicles paired to the same phone?
